### PR TITLE
Fix bug about criteo download notebook

### PR DESCRIPTION
**Pull request by bschifferer**
Fixing bug related to criteo download

I screwed up the branch by commiting wrong files. I created the PR again here: https://github.com/NVIDIA-Merlin/NVTabular/pull/1453

Please delete PR after 1453 is merged

### CODE REVIEW

File: examples/scaling-criteo/01-Download-Convert.ipynb [hunk 1]
```diff
@@ -55,7 +55,7 @@
     "from nvtabular.utils import download_file\n",
     "\n",
     "download_criteo = True\n",
-    "BASE_DIR = os.environ.get(\"BASE_DIR\", \"/raid/data/criteo\")\n",
+    "BASE_DIR = os.environ.get(\"BASE_DIR\", \"/raid/benny/11_criteo\")\n",
```
Comment (jperez999) [inline examples/scaling-criteo/01-Download-Convert.ipynb:58]: can you put this back to the default /raid/data/criteo ?

File: examples/scaling-criteo/01-Download-Convert.ipynb [hunk 2]
```diff
@@ -69,22 +69,24 @@
...
-      "text/plain": [
-       "downloading day_1.gz: 0.00B [00:00, ?B/s]"
-      ]
-     },
-     "metadata": {},
-     "output_type": "display_data"
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "downloading day_0.gz: 16.3GB [30:56, 8.79MB/s]                                  \n",
```
Comment (jperez999) [inline examples/scaling-criteo/01-Download-Convert.ipynb:79]: Do we want to change this output @benfred ?

File: examples/scaling-criteo/01-Download-Convert.ipynb [hunk 3]
```diff
@@ -189,9 +191,17 @@
...
-   "execution_count": 9,
+   "execution_count": 6,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "distributed.preloading - INFO - Import preload module: dask_cuda.initialize\n"
```
Comment (jperez999) [inline examples/scaling-criteo/01-Download-Convert.ipynb:201]: did this come from a change you made or is this just a cell you ran and we captured the output by mistake.

**Review by jperez999 — CHANGES_REQUESTED** [review]

File: examples/scaling-criteo/01-Download-Convert.ipynb (hunk 3 above)
Comment (bschifferer) [reply examples/scaling-criteo/01-Download-Convert.ipynb:201]: This are INFO notification, when I initalize the CUDA Cluster

image

File: examples/scaling-criteo/01-Download-Convert.ipynb (hunk 2 above)
Comment (bschifferer) [reply examples/scaling-criteo/01-Download-Convert.ipynb:79]: @benfred what is your opinion? I am not even sure what the related issue could be?

**Comment by bschifferer** [comment]
I screwed up the branch by commiting wrong files. I created the PR again here: https://github.com/NVIDIA-Merlin/NVTabular/pull/1453

Please delete PR after 1453 is merged

**Pull request closed without merging**